### DISCUSSION TITLE
Install on Raspberry Pi 4 with sudo wget

### DISCUSSION THREAD

**Discussion by hossihub** [philippe44/AirConnect#503] [Q&A] [answered]
Could someone provide the correct url to get the file for Raspberry Pi 4? tried the following but cant get it to work

sudo wget https://raw.githubusercontent.com/philippe44/AirConnect/master/bin/aircast-linux-aarch64

**Comment by philippe44** [comment, marked as answer]
This answer is in the updated README: all files add in the zip archive that you have to download as a whole and then extract the one you want

**Reply by hossihub** [reply]
Yes sorry found out how just after. Works like a charm.

**Reply by philippe44** [reply]
no worries
